lib/dealership.rb

```diff
@@ -0,0 +1,53 @@
+class Dealership
+  attr_reader :name, :address, :inventory, :car
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    inventory.count
+  end
+
+  def add_car(car)
+    inventory << car
+  end
+
+  def has_inventory?
+    return false if inventory.count == 0
```

Comment:
This returns false, it does not evaluate the `if inventory.count == 0`. Plus, while it is readable, it is not readable by Ruby in the way you expect. I suggest writing what you wrote here as: 
```
if inventory.count == 0
     return false
else 
     return true
end
```

You only wrote one test to check for a `false` inventory, which passes. If you wrote a test that returned `true` for inventory, you would be forced to rewrite this functionality as it does not ever pass for a `true` test.